react/ActionMenu/Readme.md

```diff
@@ -114,65 +115,13 @@ const anchorRef = React.createRef();
   {state.menuDisplayed &&
     <ActionMenu
       anchorElRef={anchorRef}
-      placement="bottom-end"
-      buttonRef={testRef}
+      popperOptions={{ placement: 'bottom-end'}}
       onClose={hideMenu}>
       <ActionMenuItem left={<Icon icon='file' />}>Item 1</ActionMenuItem>
   </ActionMenu>}
 </div>
 ```
 
-### ActionMenu inside custom fixed elements
-
-The ActionMenu component is rendered at the root of the DOM tree using a [Portal](https://reactjs.org/docs/portals.html) and a fixed z-index. Since Modals for example have a higher z-index than ActionMenus, an ActionMenu _inside_ a Modal would be displayed behind it, instead of over it. To fix this problem, the Popper instance
-is given a reference to the Modal DOM node for its `container` option via a special
-React context.
-
-Any z-indexed element that will display action menus should use this technique to
-provide a ref to a node inside this stacking context (CSS context, not React
-context).
-
-```
-import { useRef, useState, useCallback } from 'react'
-import PopperContainerContext from '../PopperContainerContext'
-import ActionMenu, { ActionMenuItem, ActionMenuHeader } from './index';
-import Icon from '../Icon';
-
-const MyActionMenu = ({ anchorRef, buttonRef, onClose }) => {
-  return <ActionMenu
-      anchorElRef={anchorRef}
-      placement="bottom-end"
-      buttonRef={buttonRef}
-      onClose={onClose}>
-      <ActionMenuItem onClick={onClose} left={<Icon icon='file' />}>Item 1</ActionMenuItem>
-  </ActionMenu>
-}
-
-const MyFixedElement = () => {
-  const ref = useRef()
-  const [shown, setShown] = useState(isTesting())
-  const buttonRef = useRef()
-  const show = useCallback(() => setShown(true), [setShown])
-  const hide = useCallback(() => setShown(false), [setShown])
-  return <PopperContainerContext.Provider value={ref}>
-    <div style={{ position: 'fixed', height: 100, width: 100, bottom: 0, top: 0, margin: 'auto', left: 0, right: 0, zIndex: 1 }}>
-      <div ref={ref} />
-      <button onClick={show} ref={buttonRef}>Open action menu</button>
-      {shown ? <MyActionMenu onClose={hide} buttonRef={buttonRef}/> : null}
-    </div>
-  </PopperContainerContext.Provider>
-};
-
-initialState = { shown: isTesting() };
-
-<>
-  <button onClick={() => setState({ shown: !state.shown })}>
-    Toggle custom fixed element
-  </button>
-  { state.shown && <MyFixedElement /> }
-</>
-```
-
 ### ActionMenu within modals
```

Comment:
it seems we still have a paragraph about `containerElRef`. Should we remove it too?

Reply:
Oh snap good catch ! I left it there to check that there was nothing broken in the screenshots, but yeah it should be removed now 💯